### ISSUE TITLE
Why are some users in the test set not present in the training set?

### ISSUE THREAD

**Issue by PassengerInDreams** [XuChen0427/FairDiverse#2]

**Comment by PassengerInDreams** [comment]
> No description provided.
